dashboard/app/models/pd/application/teacher_application_base.rb

```diff
@@ -65,11 +65,13 @@ def update_user_school_info!
     end
 
     def update_scholarship_status(scholarship_status)
-      Pd::ScholarshipInfo.update_or_create(user, application_year, scholarship_status)
+      Pd::ScholarshipInfo.update_or_create(user, application_year, course_name, scholarship_status)
```

Comment:
Is `course_name` a user-friendly string (e.g. `"CS Discoveries"`) or a short code (e.g. `"csd"`)?

Reply:
user-friendly, so it matches the `course` we store for workshops.

Reply:
but based on your feedback [here](https://github.com/code-dot-org/code-dot-org/pull/28284#issuecomment-488107680) I'm going to switch to the simplfied course code ('csd')